### ISSUE TITLE
Why is the output of the model NaN when running train.py?

### ISSUE THREAD

**Issue by AlexZzy1985** [richard-peng-xia/LMPT#8]
When i run the follwoing commant: python lmpt/train.py \
--dataset 'voc-lt' \
--seed '0' \
--pretrain_clip 'ViT16' \
--batch_size 64 \
--epochs 50 \
--class_token_position 'end' \
--ctx_init '' \
--n_ctx 16 \
--m_ctx 2 \
--training_method 'lmpt' \
--lr 5e-4 \
--loss_function dbl \
--cseloss softmargin \
--optimizer sgd \
--neg_scale 2.0 \
--gamma 0.2 \
--lam 0.5
the output of 'CustomCLIP' is NaN. I've carefully checked the code and found that in CustomCLIP, there are NaNs in the result of prompts = self.prompt_learner(). I'm wondering if there are any friends who have encountered similar problems. I'm really looking forward to getting your help. Also, thanks to the author for the outstanding work!
